docs/docs/setup/update.md

```diff
@@ -4,4 +4,85 @@ sidebar_position: 9
 
 # Update Tobira
 
-TODO
+Updating Tobira is super simple in theory:
+replace the old binary with the new one and restart the processes.
+But due to breaking changes and other reasons, there are a few things to look out for.
+
+*Note*: this document is fairly long, but only to cover lots of cases that rarely occur.
+Usually, you can skip most of this.
+
+
+## Preparation
+
+Read the release notes of all releases between your current Tobira version and the one you want to update to.
+Most importantly, check all [breaking changes](../versioning-policy) and act upon those as required.
+**Reminder**: the version labels are purely cosmetic and minor version bumps can have breaking changes!
+
+Deal with some of the possible breaking changes:
+- Update your Opencast to a version that will also work with the new Tobira.
+- Make sure your Opencast satisfies all requirements of the new Tobira.
+- Update your PostgreSQL to a version compatible with the new Tobira.
+- Adjust your authentication integration if Tobira's auth system changed.
+
+:::caution
+Now is a good time to perform a database backup.
+Of course, database backups are already created automatically regularly, right?
+*Right?*
+:::
+
+
+## Make changes
+
+- **Replace the old Tobira binary by the new one**
+- **Update `config.toml`**
+    - We recommend doing this regardless of whether there was a breaking configuration change:
+      it makes sure the config file stays as close to the upstream one as possible, making future updates easier.
+    - *Note*: Tobira does not watch nor automatically reload the config file on change.
+      Thus, you can change the file in place: it only takes effect once you restart the process.
+    - This is best done via *3-way merge*. For example, when updating from v1.3 to v1.4:
+        - "base" is the original `config.toml` for v1.3 (attached to the release).
+        - "left" is the original `config.toml` for v1.4 (attached to the release).
+        - "right" is your current `config.toml` that's deployed on the server.
+        - There are mutliple 3-way merge tools you can use.
+- *Conditional*: if required, update other files referenced by `config.toml`.
+- *Conditional*: if there were breaking Tobira CLI changes, update service files, scripts and other places as required.
+
+## *Conditional*: Update MeiliSearch
```

Comment:
It might be slightly confusing that this section follows the section where you update Tobira proper, but then you say you might have to do this before that. :thinking:

Reply:
Fair. :shrug: